src/gui/aboutdialog.cpp

```diff
@@ -88,13 +88,21 @@ AboutDialog::AboutDialog(QWidget *parent)
         licensefile.close();
     }
 
-    // Libraries
+    // Software Used
     m_ui->labelQtVer->setText(QT_VERSION_STR);
     m_ui->labelLibtVer->setText(Utils::Misc::libtorrentVersionString());
     m_ui->labelBoostVer->setText(Utils::Misc::boostVersionString());
     m_ui->labelOpensslVer->setText(Utils::Misc::opensslVersionString());
     m_ui->labelZlibVer->setText(Utils::Misc::zlibVersionString());
 
+    const QString DBIPText = QString(
+                                 "<html><head/><body><p>"
+                                 "%1"
+                                 " (<a href=\"https://db-ip.com/\">https://db-ip.com/</a>)</p></body></html>")
+                             .arg(tr("The free IP to Country Lite database by DB-IP is used for resolving the countries of peers. "
+                                     "The database is licensed under the Creative Commons Attribution 4.0 International License"));
```

Comment:
nitpick, I think you forgot the ending dot after "License".

Reply:
Not really. After that there is this text(notice space): ` (https://db-ip.com/)`